circuits/ts/__tests__/IncrementalQuinTree.test.ts

```diff
@@ -15,56 +13,71 @@ describe("IncrementalQuinTree circuit", function test() {
   const leavesPerNode = 5;
   const treeDepth = 3;
 
-  let circuitLeafExists: tester.WasmTester;
-  let circuitGeneratePathIndices: tester.WasmTester;
-  let circuitQuinSelector: tester.WasmTester;
-  let splicerCircuit: tester.WasmTester;
+  let circuitLeafExists: WitnessTester<["leaf", "path_elements", "path_index", "root"]>;
+  let circuitGeneratePathIndices: WitnessTester<["in"], ["out"]>;
+  let circuitQuinSelector: WitnessTester<["in", "index"], ["out"]>;
+  let splicerCircuit: WitnessTester<["in", "leaf", "index"], ["out"]>;
 
   before(async () => {
-    circuitLeafExists = await tester.wasm(
-      path.resolve(__dirname, "../../circom/test", `quinTreeLeafExists_test.circom`),
-    );
-    circuitGeneratePathIndices = await tester.wasm(
-      path.resolve(__dirname, "../../circom/test", `quinGeneratePathIndices_test.circom`),
-    );
-    circuitQuinSelector = await tester.wasm(path.resolve(__dirname, "../../circom/test", `quinSelector_test.circom`));
-    splicerCircuit = await tester.wasm(path.resolve(__dirname, "../../circom/test", `splicer_test.circom`));
+    circuitLeafExists = await circomkit.WitnessTester("quinLeafExists", {
+      file: "./trees/incrementalQuinTree",
+      template: "QuinLeafExists",
+      params: [3],
+    });
```

Comment:
Just a question, I saw there is another circom called `incrementalMerkleTree`, and there was tests toward it before. Does it need to be tested?

Reply:
not sure I understand what you mean here, could you rephrase please

Reply:
I mean the `incrementalMerkleTree.circom` seems not be tested, will it be tested anywhere or we don’t need it?

Reply:
Oh right, yea this was not tested even before, and it's not even used actually. We always use the quinary one as that's what we use throughout the codebase